src/components/PostEditor/PostEditor.js

```diff
@@ -447,19 +447,26 @@ class PostEditor extends React.Component {
       this.state.post,
       postUpdates
     )
-    const { isEvent, isProject, isProposal } = this.props
+
+    let validEvent = type?.length > 0
```

Comment:
shouldn't this be called validPost?

Reply:
Ah, yikes ... This deals with post type data only (there are other tests for valid post), so I renamed it validTypeData.